src/components/Country/Country.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+import { Link } from 'react-router-dom';
+import style from './Country.module.scss';
+
+const Country = ({ name, flag }) => (
+  <article>
+    <h2>
+      <Link to={name}>{name}</Link>
+    </h2>
+    <img src={flag} alt="Country flag not available" className={style.flag} />
+  </article>
+);
+
```

Comment:
- [ ] Good job but :+1:  but please make sure that on your home page we have a list of categories that could be filtered using given parameters, e.g. category name as required for this project. You can see check the screenshots below to  see an implementation :arrow_down: 

### Filter implementation
![Screenshot from 2022-10-22 20-41-01](https://user-images.githubusercontent.com/6615394/197361668-e49e5517-d19e-4ba0-8b32-c0f82df61cef.jpg)
![Screenshot from 2022-10-22 20-41-09](https://user-images.githubusercontent.com/6615394/197361697-52608bb2-bed5-45d4-a6f3-585ba87614cd.jpg)